fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -1790,7 +1790,7 @@ public void testCreateResourceWithoutContentType() {
     public void testUpdateObjectWithoutContentType() throws IOException {
         final HttpPut httpPut = new HttpPut(getLocation(postObjMethod()));
         // use a bytestream-based entity to avoid settin a content type
-        httpPut.setEntity(new ByteArrayEntity("bogus content".getBytes()));
+        httpPut.setEntity(new ByteArrayEntity("bogus content".getBytes("utf8")));
```

Comment:
I would recommend using `java.nio.charset.StandardCharsets.UTF_8` instead of a `String` here.